core/shared/src/main/scala/zio/internal/FiberRuntime.scala

```diff
@@ -987,6 +987,16 @@ final class FiberRuntime[E, A](fiberId: FiberId.Runtime, fiberRefs0: FiberRefs,
 
       ops += 1
 
+      if (RuntimeFlags.opLog(_runtimeFlags)) {
+        val message = ZIO.render(cur)
+        log(
+          message = () => message,
```

Comment:
Ideally we only eval `render` when logging is enabled.
```suggestion
        log(
          message = () => ZIO.render(cur),
```